functorch/_src/decompositions.py

```diff
@@ -489,6 +491,16 @@ def addmm(self: Tensor, mat1: Tensor, mat2: Tensor, beta: int = 1, alpha: int =
     return beta * self + out
 
 
+@register_decomposition(aten.l1_loss_backward)
+def l1_loss_backward(grad_output: Tensor, self: Tensor, target: Tensor, reduction: int = Reduction.MEAN.value):
+    assert(reduction in tuple(red.value for red in Reduction))
+    sign = torch.sign(self - target)
+
+    # numel doesn't work as the denominator if self is non-contiguous
```

Comment:
Why not?

Reply:
Oh interesting. I thought I was seeing different results from playing around with it in Python but I think I must have messed something up locally. Switching back